.github/workflows/publish-packages.yml

```diff
@@ -0,0 +1,44 @@
+name: Publish Packages
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  checks:
+    uses: ./.github/workflows/checks.yml
+
+  publish:
+    needs: checks
+    runs-on: ubuntu-latest
+    environment: NPM
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Setup Node
+      uses: actions/setup-node@v3
+      with:
+        node-version: 16.x
+        cache: yarn
+        registry-url: https://registry.npmjs.org
+
+    - name: Configure Git user
+      run: |
+        git config user.email "<>"
+        git config user.name "Github Actions Bot"
+
+    - name: Install dependencies
+      run: yarn --frozen-lockfile
```

Comment:
Since we have set `useWorkspaces: true` Lerna will delegate bootstrapping to yarn workspaces, so this is essentially no different to `lerna bootstrap`. I'm using `yarn` directly here to avoid installing `lerna` first.